src/main/java/org/team1540/robot2018/OI.java

```diff
@@ -43,119 +43,137 @@ private static double scale(double input, double pow) {
   private static Joystick driver = new Joystick(0);
   private static Joystick copilot = new Joystick(1);
 
-  public static final int X = 3;
-  public static final int Y = 4;
+  // Buttons
   public static final int A = 1;
   public static final int B = 2;
+  public static final int X = 3;
+  public static final int Y = 4;
 
-  public static final int BACK = 7;
-  public static final int START = 8;
   public static final int LB = 5;
   public static final int RB = 6;
+  public static final int BACK = 7;
+  public static final int START = 8;
 
-  // auto intake
-  static Button copilotLB = new JoystickButton(copilot, LB);
-  // auto eject
-  static Button copilotRB = new JoystickButton(copilot, RB);
+  // Joysticks
+  public static final int LEFT_X = 0;
+  public static final int LEFT_Y = 1;
+  public static final int LEFT_TRIG = 2;
+  public static final int RIGHT_TRIG = 3;
+  public static final int RIGHT_X = 4;
+  public static final int RIGHT_Y = 5;
 
+  // TODO: Remove unused button
   // wrist to forward 45 degrees
-  static Button copilotX = new JoystickButton(copilot, X);
-  // wrist to full down
-  static Button copilotA = new JoystickButton(copilot, A);
-  // wrist to backward 45 degrees
-  static Button copilotY = new JoystickButton(copilot, Y);
-  // wrist to full back
-  static Button copilotB = new JoystickButton(copilot, B);
-
-  // to max height + wrist up
-  static Button copilotDPadUp = new DPadButton(copilot, 0, DPadAxis.UP);
-  // to lower scale height
-  static Button copilotDPadDown = new DPadButton(copilot, 0, DPadAxis.DOWN);
-  // to max scale height (no wrist)
-  static Button copilotDPadLeft = new DPadButton(copilot, 0, DPadAxis.LEFT);
-  // to switch height
-  static Button copilotDPadRight = new DPadButton(copilot, 0, DPadAxis.RIGHT);
-
-  // manual tape out
-  static Button copilotBack = new JoystickButton(copilot, BACK);
-  // manual tape in
-  static Button copilotStart = new JoystickButton(copilot, START);
-
-  static Button elevatorJoystickActivation = new Button() {
-    @Override
-    public boolean get() {
-      return Utilities.processDeadzone(copilot.getRawAxis(1), Tuning.joystickWristLiftDeadzone)
-          != 0; // zero values mean it's within the deadzone
-    }
-  };
+  // static Button copilotX = new JoystickButton(copilot, X);
 
-  static Button wristJoystickActivation = new Button() {
+  // ELEVATOR
+  public static double getElevatorAxis() {
+    return scale(Utilities.processDeadzone(copilot.getRawAxis(LEFT_Y), Tuning.axisDeadzone), 2);
+  }
+
+  // TODO: Add a deadzone button to ROOSTER
+  // TODO: Add something (function/interface) for disabling a command when joystick is not being used
```

Comment:
Probably should create issues for these.

Reply:
2fancy4me